document.go

```diff
@@ -116,8 +117,12 @@ func (d *Document) MarshalJSON() ([]byte, error) {
 // JSONSchema returns a jsonschema.Schema instance.
 func (Document) JSONSchema() *jsonschema.Schema {
 	return &jsonschema.Schema{
-		Type:        "object",
-		Title:       "Document",
-		Description: "Contents of the envelope that must contain a $schema.",
+		Type:  "object",
+		Title: "Document",
+		Description: here.Doc(`
+			Contains the document payload to be included inside an Envelope.
+			
+			The document must contain a <code>$schema</code> property that identifies
```

Comment:
I wonder whether we could enter markdown here directly. We don't even need to transform it to html, as it will end up in a MDX file that will take care of that.

Reply:
Yeah, let's try with the markdown.

Reply:
I think I found a way, a bit messy, but works :-)